src/bitmessageqt/__init__.py

```diff
@@ -3873,7 +3873,7 @@ def tableWidgetInboxItemClicked(self):
         else:
             data = self.getCurrentMessageId()
             if data != False:
-                message = "Error occurred: could not load message from disk."
+                message = "Error occurred: could not load message from disk." #### TODO: _translate( this ! happens frequently
```

Comment:
please put the TODO in the line before the message.

Reply:
Just drop this patch. I've already marked this string for translation and also completely rewritten the method.